go.mod

```diff
@@ -5,28 +5,31 @@ go 1.12
 require (
 	github.com/Jeffail/gabs v1.1.1
 	github.com/ant31/crd-validation v0.0.0-20180801212718-38f6a293f140 // indirect
-	github.com/coreos/prometheus-operator v0.0.0-00010101000000-000000000000
-	github.com/emicklei/go-restful v2.8.0+incompatible // indirect
+	github.com/coreos/prometheus-operator v0.32.1-0.20191022141338-971190798bb6
```

Comment:
v0.32? But prometheus-operator is already at v0.33 :man_shrugging:

Reply:
No clue, this is what go modules does when I do `go get github.com/coreos/prometheus-operator@971190798bb6bafcd6c109dfa344be4458c39018`

vendored code is correct, I checked.

Reply:
If I remove this, and run go get, then go modules complains and says that there is no version `v00000000000`